alembic/versions/936887038ba8_sync_cg_and_statusdb.py

```diff
@@ -0,0 +1,121 @@
+"""Sync cg and statusdb
+
+Revision ID: 936887038ba8
+Revises: 5cb3ce4c3e39
+Create Date: 2023-09-14 09:26:50.748369
+
+"""
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "936887038ba8"
+down_revision = "5cb3ce4c3e39"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.alter_column(
+        "application",
+        "prep_category",
+        existing_type=mysql.ENUM("cov", "mic", "rml", "tgs", "wes", "wgs", "wts"),
+        nullable=False,
+    )
+    op.alter_column(
+        "family",
+        "priority",
+        existing_type=mysql.ENUM("research", "standard", "priority", "express", "clinical_trials"),
+        nullable=False,
+    )
+    op.alter_column("invoice", "customer_id", existing_type=mysql.INTEGER(), nullable=False)
+    op.alter_column("invoice", "comment", existing_type=mysql.TEXT(), nullable=True)
+
+    op.drop_index("customer_invoice_pk", table_name="invoice")
```

Comment:
How come we are dropping this index?

Reply:
It is basically a conversion with the line below. In the Python model we have this as a foreign key but in the actual database we have it as an index. Foreign keys are automatically indexed, though, so the line beneath should ensure that we have an index and a foreign key on the column. 

At the moment there would be no constraint to going into the db manually and an entry in the invoice with a customer_id which does not exist.

Reply:
I will resolve this conversation.